js/app.js

```diff
@@ -13,6 +13,8 @@ var Tracker = function () {
 	this.photoArray = [];
 }
 
+var tracker = new Tracker();
```

Comment:
Will you copy and paste your prefs in slack or an email for me to take a look at? I think it’s missing or has one too many commas somewhere…

From:  Isabella Organ
Reply-To:  isabellaorgan/vote_tracker
Date:  Thursday, September 3, 2015 at 3:32 PM
To:  isabellaorgan/vote_tracker
Cc:  Scott Schmidt
Subject:  Re: [vote_tracker] photos click, work on tally (#4)


In js/app.js:
> @@ -13,6 +13,8 @@ var Tracker = function () {
>  	this.photoArray = [];
>  }
>  
> +var tracker = new Tracker();
Yep, I just tried this and I still have a 4 space tab. And as soon as I save the Preferences, I get the following error:
Error trying to parse settings: Unexpected trailing characters in ~/Library/Application Support/Sublime Text 2/Packages/User/Preferences.sublime-settings:10:1
But I can't find any trailing characters.
I'll keep working on it though! I'll just use the Googles!

—
Reply to this email directly or view it on GitHub.